testsuite/test-generate-alluxio.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+source test-generate-common.sh
+source test-common.sh
+source test-config.sh
+
+GenerateAlluxioStandardTests() {
```

Comment:
in general could testing against specific versions of alluxio be done?  see `test-common.sh` to see how its setup and there's some parts in `test-generate.sh` and you can see the versioning basics in `test-generate-spark.sh`

Reply:
Okay I will check versions testing and add it.